### DISCUSSION TITLE
Load RDF4J custom SPARQL function in GraphDB?

### DISCUSSION THREAD

**Discussion by edmondchuc** [eclipse-rdf4j/rdf4j#4612] [Q&A]
Has anyone tried getting an RDF4J custom SPARQL function to work in GraphDB? I can't seem to get GraphDB to detect it.

Here's my [StackOverflow question](https://stackoverflow.com/questions/76397949/graphdb-load-rdf4j-custom-sparql-function-in-docker-image) for more information on how I'm loading it in GraphDB with a Docker image.

Thanks in advance.

**Comment by hmottestad** [comment]
It's the weekend so I don't think Ontotext has had a chance to respond to your Stack Overflow question. I would recommend giving them a few days to answer, and if you don't hear anything you can try contacting them on twitter or by email: https://graphdb.ontotext.com/documentation/10.2/support.html
